docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -2,55 +2,65 @@
 using System.Collections.Generic;
 using Microsoft.ML;
 using Microsoft.ML.Data;
-using static Microsoft.ML.Transforms.OneHotEncodingEstimator;
+using Microsoft.ML.Transforms;
 
-namespace Samples.Dynamic
+namespace Samples.Dynamic.Transforms.Categorical
 {
     public static class OneHotEncoding
     {
         public static void Example()
         {
-            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
-            // as well as the source of randomness.
+            // Create a new ML context, for ML.NET operations. It can be used for
+            // exception tracking and logging, as well as the source of randomness.
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable.
-            var samples = new List<DataPoint>()
+            var samples = new List<DataPoint>
             {
-                new DataPoint(){ Education = "0-5yrs" },
-                new DataPoint(){ Education = "0-5yrs" },
-                new DataPoint(){ Education = "6-11yrs" },
-                new DataPoint(){ Education = "6-11yrs" },
-                new DataPoint(){ Education = "11-15yrs" },
+                new DataPoint {Education = "0-5yrs"},
+                new DataPoint {Education = "0-5yrs"},
+                new DataPoint {Education = "6-11yrs"},
+                new DataPoint {Education = "6-11yrs"},
+                new DataPoint {Education = "11-15yrs"}
             };
 
             // Convert training data to IDataView.
-            var data = mlContext.Data.LoadFromEnumerable(samples);
+            IDataView data = mlContext.Data.LoadFromEnumerable(samples);
 
             // A pipeline for one hot encoding the Education column.
-            var pipeline = mlContext.Transforms.Categorical.OneHotEncoding("EducationOneHotEncoded", "Education");
+            OneHotEncodingEstimator pipeline =
+                mlContext.Transforms.Categorical.OneHotEncoding(
+                    "EducationOneHotEncoded","Education");
```

Comment:
nit: space

Reply:
CTR+K, CTR+D

---
In reply to: [278758039](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278758039) [](ancestors = 278758039)

Reply:
(meaning, add a space after the comma between the column names)

---
In reply to: [278759192](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278759192) [](ancestors = 278759192,278758039)